web/blueprints/facilities/tables.py

```diff
@@ -104,3 +105,22 @@ class PatchPortRow(BaseModel):
     switch_port: LinkColResponse | None = None
     edit_link: BtnColResponse
     delete_link: BtnColResponse
+
+
+class RoomTenanciesTable(BootstrapTable):
+    _render_toolbar = False
+
+    inhabitant = BtnColumn("Bewohner")
+    begins_at = DateColumn("Von")
+    ends_at = DateColumn("Bis")
+    status = Column("Status")
+
+    class Meta:
+        table_args = {"data-sort-name": "begins_at"}
+
+
+class RoomTenanciesRow(BaseModel):
+    inhabitant: BtnColResponse
```

Comment:
Why not `BtncolResponse | None` and allow showing tenancies with no users? Seems like most of the time this should not happen but if it does we would want to see it.

Reply:
@FestplattenSchnitzel any reason you resolved this and all the other comments of mine? were they not useful, or did this happen on accident?

Reply:
BTW a use case for this would be: there is a tenancy with a user (i.e. `person_id`), but the person id is not mapped to any user in our DB, e.g. because they did not register with us yet.

Reply:
@lukasjuhrich I think I resolved the suggestions b/c they were implemented back then (https://github.com/agdsn/pycroft/commit/ede44d9578ad63d5c7430f373cfec4204679dc1c), but looks like you overrode the changes?